<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAEquationRampWidget</class>
 <widget class="QWidget" name="GTAEquationRampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>517</width>
    <height>492</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>517</width>
    <height>492</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="ImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>495</width>
       <height>389</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>495</width>
       <height>389</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/images/forms/img/RampGraph_EndValue.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>495</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>495</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="rampLB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rise Time (s):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="riseTimeLE"/>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="SearchTB2">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="startLE"/>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="SearchTB">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="endLE"/>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="SearchTB3">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="startLB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Start Value:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="EndTimeCB">
        <item>
         <property name="text">
          <string>End Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Offset</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
